### PR TITLE
feat(dws): add new datasource to query parameters of the cluster

### PR DESCRIPTION
**Pull request by wuzhuanhong**
**What this PR does / why we need it**:
Add new dataSource to query parameters under specified the cluster.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add new data source.
2. add corresponding to document and acceptance test.
```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST=./huaweicloud/services/acceptance/dws TESTARGS='-run TestAccDataSourceClusterParameters_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccDataSourceClusterParameters_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceClusterParameters_basic
=== PAUSE TestAccDataSourceClusterParameters_basic
=== CONT  TestAccDataSourceClusterParameters_basic
--- PASS: TestAccDataSourceClusterParameters_basic (17.72s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       17.780s```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    <!-- If the resource depends the parent resource(s), please provide the related check result(s) of the CheckDeleted validation.
    ab. Related resources (parent resources) not found
    \>>>>>> Paste the screenshot here <<<<<<
    -->
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    <!-- If the resource depends the parent resource(s), please provide the related check result(s) of the CheckDeleted validation.
    bb. Related resources (parent resources) not found
    \>>>>>> Paste the screenshot here <<<<<<
    -->

### CODE REVIEW

**Comment by wuzhuanhong** [comment]
make testacc TEST=./huaweicloud/services/acceptance/dws TESTARGS='-run TestAccDataSourceClusterParameters_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccDataSourceClusterParameters_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceClusterParameters_basic
=== PAUSE TestAccDataSourceClusterParameters_basic
=== CONT  TestAccDataSourceClusterParameters_basic
--- PASS: TestAccDataSourceClusterParameters_basic (17.79s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       17.844s

File: docs/data-sources/dws_cluster_parameters.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+* `restart_required` - Whether the DWS cluster needs to be restarted after modifying the parameter value.
+
+* `description` - The description of the parameter.
+
+<a name="parameters_values_struct"></a>
+The `values` block supports:
+
+* `type` - The type of the parameter.
+  + **cn**
+  + **dn**
```
Comment (profoundwu) [inline docs/data-sources/dws_cluster_parameters.md:73]: The value can be **cn** or **dn**

**Pull request merged**